test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -33,6 +33,22 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task TagHelper_InvalidIndexerDoesNotFail()
```

Comment:
This should have a comment linking to the issue and explaining what's being tested and how. It's totally non-obvious.